packages/apps/docs/src/docs/deploy/chainweb/compact-a-database.md

```diff
@@ -1,103 +1,121 @@
      -h,--help                Show this help text
-
-   Available commands:
-     ea                       Generate Chainweb genesis blocks and their payloads
-     run-nodes                Run a local cluster of chainweb-node binaries
-     slow-tests               Run slow Chainweb tests
-     tx-list                  List all transactions in a chain starting with the most recent block
-     genconf                  Interactively generate a chainweb-node config
-     header-dump              Dump Block Headers to a JSON array
-     b64                      Command line utlis for Chainweb base64 encode/decode
-     db-checksum              Generate a checksum of all the checkpointer database tables between an inclusive range of blocks.
-     known-graphs             Encode know graphs as JSON values
-     tx-sim                   Simulate tx execution against real pact dbs
-     compact                  Compact pact database
-     pact-diff                Diff the latest state of two pact databases
-     calculate-release        Calculate next service date and block heights for upgrades
    ```
 
-3. Create a backup copy of your current pact `sqlite` database to save all current state by running a command similar to the following:
+3. Compact your `rocksdb` and `sqlite` databases by running the `compact` command with the following arguments:
+
+   - `--from` to specify the path to the database directory you want to compact. You should specify the database root directory that contains the `0/sqlite` and `0/rocksdb` subdirectories. For example, the `data/state/chainweb/db` directory is the root directory for the `data/state/chainweb/db/0/sqlite` directory and the `data/state/chainweb/db/0/rocksdb` directory.
+   - `--to` to specify the path to the compacted database. The compact program writes the compacted databases to the `$DIR/0/sqlite` and `$DIR/0/rocksdb` subdirectories within the directory you specify.
+   - `--log-dir` to specify the directory where you want the `compact` program to put the log files it creates, one for each chain. If the directory doesn’t exist, the `compact` program creates it. These logs can be useful for debugging if something goes wrong.
+   - `--chainweb-version` to specify the network identifier for the node. This argument is optional if you're compacting a database for the `mainnet01` network. If you're compacting a database for another network—for example, the Kadena test network—you must specify the network identifier. Valid values are "`development`", "`testnet04`", and "`mainnet01`".
+
+   For example, if you are using the default location for the database directory and a node connected to the Kadena test network, run a command similar to the following:
 
    ```bash
-   cp -r /data/state/chainweb/db/0/sqlite /tmp/sqlite-backup-dd-mm-yyyy
+   compact --from ~/.local/share/chainweb-node/testnet04 --to ~/.local/share/chainweb-node/compact-db --log-dir /tmp/compaction-log-files
```

Comment:
I just realised that, because in this example you are compacting testnet (as signified by the directory), this would fail without `--chainweb-version testnet04` as a flag.